packages/design/src/FormManager/FormEdit/components/DraggableList.test.tsx

```diff
@@ -0,0 +1,7 @@
+/**
+ * @vitest-environment jsdom
+ */
+import { describeStories } from '../../../test-helper';
+import meta, * as stories from './DraggableList.stories';
+
+describeStories(meta, stories);
```

Comment:
Thanks for adding this, looks great. fyi, from a discussion in TTS Slack, I just learned about this - https://vitest.dev/guide/browser/

Looks like it might be a good drop-in alternative for jsdom that we could consider trying.